lib/wallet/src/Cardano/Wallet/Write/ProtocolParameters.hs

```diff
@@ -1,5 +1,7 @@
+{-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE UndecidableInstances #-}
```

Comment:
😐 I think this should be removable with https://input-output.atlassian.net/browse/ADP-2353